app/Client/Client.php

```diff
@@ -45,13 +45,13 @@ public function share(string $sharedUrl, array $subdomains = [])
         $sharedUrl = $this->prepareSharedUrl($sharedUrl);
 
         foreach ($subdomains as $subdomain) {
-            $this->connectToServer($sharedUrl, $subdomain, config('expose.auth_token'));
+            $this->connectToServer($sharedUrl, $subdomain, $this->configuration->auth());
```

Comment:
This btw will enable the --auth option to work too. Before always the config was used, never the auth-option.